### PR TITLE
feat: :sparkles: Add more info on the KDA Staking tab

### PR DESCRIPTION
**Pull request by saviojks**

### CODE REVIEW

File: public/locales/en/assets.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "Min Epochs To Claim": "Minimum Epochs To Claim",
     "Min Epochs To Unstake": "Minimum Epochs To Unstake",
     "Min Epochs To Withdraw": "Minimum Epochs To Withdraw",
+    "Qtd_Holders": "Qtd Holders",
```
Comment (lucasportella) [inline public/locales/en/assets.json:42]: shot not be in Qty or Quantity?

File: public/locales/pt-BR/assets.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "Min Epochs To Claim": "Épocas Mínimas para Reivindicar",
     "Min Epochs To Unstake": "Épocas Mínimas para Desapostar",
     "Min Epochs To Withdraw": "Épocas Mínimas para Retirar",
+    "Qtd_Holders": "Qtd Holders",
```
Comment (lucasportella) [inline public/locales/pt-BR/assets.json:42]: same

**Review by lucasportella — CHANGES_REQUESTED** [review]

File: public/locales/pt-BR/assets.json (hunk 2 above)
Comment (lucasportella) [reply public/locales/pt-BR/assets.json:42]: here only the property, the value i think should be Quantidade de holders

File: public/locales/en/assets.json (hunk 1 above)
Comment (saviojks) [reply public/locales/en/assets.json]: Done

**Review by lucasportella — APPROVED** [review]

**Pull request merged**